test/meal-test.js

```diff
@@ -0,0 +1,14 @@
+const assert = require('chai').assert;
+const meals = require('../lib/meals.js')
+global.$ = require('jquery')(window);
```

Comment:
i believe _if_ you're using webpack to load jquery, you can simply require it here... 

```js
const $ = require('jquery')
```

Reply:
Thanks! I will try that.

Reply:
Webpack didn't seem to fix it. We had a JS questions workshop and apparently this has been an ongoing issue with no easy fix, so I'll put it on hold for now.